src/lib.rs

```diff
@@ -404,6 +441,13 @@ pub enum Operation {
     },
     /// Deleting a value
     Delete,
+    /// Deleting a range of values
+    DeleteRange {
```

Comment:
Just to discuss: For what cases we actually want to keep single `Delete` operation? Could having it can be source of performance bugs? I suppose we can delete single key with DeleteRange.

Reply:
I think we should keep `Delete` for single key deletes because it supports a bunch of things `DeleteRange` doesn't, for example:

- `WriteBatch` with transaction support (though we don't currently use this)
- `SchemaBatch::iter`s
- `SchemaBatch::merge` (which we use)

Etc

Reply:
Let's catch up about it on next daily. Maybe we will remove `Delete` in follow up PRs